models/music_rate_in_proxy.h

```diff
@@ -167,7 +167,7 @@ class music_rate_in_proxy : public DeviceNode
 
   struct Buffers_
   {
-    double data_;
+    std::vector< double > data_; //!< The buffer for incoming data
```

Comment:
@hakonsbm Based on your [previous analysis](https://github.com/nest/nest-simulator/issues/1449#issuecomment-593950033) this update makes sense, but I wonder if a smaller change, namely changing 
```c++
( *d )[ names::data ] = DoubleVectorDatum( new std::vector< double >( B_.data_ ) );
```
to
```c++
( *d )[ names::data ] = DoubleVectorDatum( new std::vector< double >( 1, B_.data_ ) );
```
so that we correctly create a single-element vector, would be the safer approach here to just fix this problem?

Reply:
Done. But we should really have a test of this model.

Reply:
Indeed, would you create a follow-up issue for a proper review of this (and other music) proxies?